orc8r/cloud/go/services/eventd/obsidian/handlers/handlers.go

```diff
@@ -53,8 +56,45 @@ const (
 	elasticFilterHardwareID = "hw_id" + dotKeyword
 	elasticFilterEventTag   = "event_tag" + dotKeyword // We use event_tag as fluentd uses the "tag" field
 	elasticFilterTimestamp  = "@timestamp"
+
+	ManageNetworkPath  = obsidian.V1Root + "networks" + obsidian.UrlSep + ":network_id"
+	LogSearchQueryPath = ManageNetworkPath + obsidian.UrlSep + "logs" + obsidian.UrlSep + "search"
+	LogCountQueryPath  = ManageNetworkPath + obsidian.UrlSep + "logs" + obsidian.UrlSep + "count"
 )
 
+// GetObsidianHandlers returns all the obsidian handlers for eventd.
+func GetObsidianHandlers() []obsidian.Handler {
+	var ret []obsidian.Handler
+
+	// Elastic
+	elasticConfig, err := config.GetServiceConfig(orc8r.ModuleName, "elastic")
+	if err != nil {
+		ret = append(ret, obsidian.Handler{Path: LogSearchQueryPath, Methods: obsidian.GET, HandlerFunc: getInitErrorHandler(err)})
+		ret = append(ret, obsidian.Handler{Path: LogCountQueryPath, Methods: obsidian.GET, HandlerFunc: getInitErrorHandler(err)})
+		ret = append(ret, obsidian.Handler{Path: EventsPath, Methods: obsidian.GET, HandlerFunc: getInitErrorHandler(err)})
+	} else {
+		elasticHost := elasticConfig.MustGetString("elasticHost")
+		elasticPort := elasticConfig.MustGetInt("elasticPort")
+
+		client, err := elastic.NewSimpleClient(elastic.SetURL(fmt.Sprintf("http://%s:%d", elasticHost, elasticPort)))
+		if err != nil {
+			ret = append(ret, obsidian.Handler{Path: LogSearchQueryPath, Methods: obsidian.GET, HandlerFunc: getInitErrorHandler(err)})
```

Comment:
minor nit: setting the error handling can be done in a loop, might be cleaner that way.